.github/workflows/inspektor-gadget.yml

```diff
@@ -504,6 +506,8 @@ jobs:
         outputs: type=registry,name=${{ steps.set-repo-determine-image-tag.outputs.container-repo }},push=true,push-by-digest=true
         cache-from: type=local,src=/tmp/.buildx-cache-new
         platforms: ${{ matrix.os }}/${{ matrix.platform }}
+        build-args: |
+          VERSION=${{ steps.set-repo-determine-image-tag.outputs.image-tag }}
```

Comment:
I wanted to keep the build-args same for both steps to avoid surprises. We are doing same of [`ig` images](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/.github/workflows/inspektor-gadget.yml#L650) so should be fin I guess!

Reply:
Good point. Let me have a look into it, if they aren't needed we should drop them from both ig-k8s and ig steps to avoid confusion!